plugins/org.eclipse.osee.ats.api/src/org/eclipse/osee/ats/api/workdef/model/WidgetDefinition.java

```diff
@@ -259,4 +259,11 @@ public void setUserGroup(IUserGroupArtifactToken userGroup) {
       this.userGroup = userGroup;
    }
 
+   public LayoutItem andEnabled(ConditionalRule... rules) {
```

Comment:
A conditional rule isn't just for enabled.  Rename to andCondition?

Reply:
Done